.github/workflows/translate.yml

```diff
@@ -0,0 +1,62 @@
+name: "Update translations"
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
In the desktop repo this is a cronjob but it can also be triggered directly.
Was the cronjob removed on purpose?